account_supplier_tax_rounding/models/account_tax.py

```diff
@@ -22,6 +28,8 @@ def compute_all(
                 quantity=quantity,
                 product=product,
                 partner=partner,
+                is_refund=False,
```

Comment:
I think it will be cleaning just to do something like.

```
if .....:
    self = self.with_context(...)
return super()
```
hopefully it just works.